src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,34 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/todo';
+import usersFromServer from '../../api/users';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  const { id, title, completed, userId } = todo;
+  const currentUser = usersFromServer.find(user => user.id === userId);
```

Comment:
The `find` method is used to locate the user associated with the todo. It's good practice to handle cases where `currentUser` might be `undefined`, especially when accessing properties like `email` and `name`. Consider adding a fallback or conditional rendering to handle such cases gracefully.